public_html/wp-content/mu-plugins/blocks/assets/src/shared/grid-layout/inspector-control.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies.
+ */
+const { Component } = wp.element;
+const { PanelBody, PanelRow, RangeControl } = wp.components;
+const { __ } = wp.i18n;
+
+const DEFAULT_SCHEMA = {
+	grid_columns: {
+		default: 2,
+		minimum: 2,
+		maximum: 4,
+	},
+};
+/**
+ * Add a slider for increasing and decreasing columns. Should be used with rest of the components in this folder. Will use and set attributes `layout` and `gird_columns`.
+ */
+class GridInspectorControl extends Component {
+
+	render() {
+		const { attributes, setAttributes } = this.props;
+		const { layout, grid_columns } = attributes;
+
+		if ( 'grid' !== layout ) {
+			return null;
+		}
+		const schema = DEFAULT_SCHEMA;
```

Comment:
[SHOULD] Since this is a shared component, this should look for values in `props` before using the static defaults, so that different values can be passed in.

Reply:
I did not added this because we only support minimum 2 and maximum 4, so it makes sense to not make this configurable.
I am marking this is as resolved #44 to make everything more manageable, but please feel free to uncheck it there if you think we should discuss this more.

Reply:
This kind of ties in with #57. If the block schema gets defined in PHP and loaded via `add_script_data`, like in the other blocks, it would be simple to use the schema values instead of a static variable. That would potentially make it DRYer as well. I'll add a comment about that over in the issue so we can close up this thread.